examples/ROOT_files/ROOT_STIR_consistency/Gate_macros/geometry.mac

```diff
@@ -31,6 +31,12 @@
 /gate/cylindricalPET/geometry/setRmin         40.5 cm
 /gate/cylindricalPET/geometry/setHeight       15.716 cm
 /gate/cylindricalPET/setMaterial              Air
+
+# Allign STIR and Gate geometries first cystals on y axis 
+# by rotate the cylindricalPET -90 degrees around z
```

Comment:
There's some spelling problems here, but it is also not quite accurate. I suggest something like this
```suggestion
# Rotate Gate geometry such that the first module will be on "top" of the scanner (i.e. x=0, y>0)
# by rotating the cylindricalPET -90 degrees around z.
# Note that STIR still needs view-offset according to half the module size.
```